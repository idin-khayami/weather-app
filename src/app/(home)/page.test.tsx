import { render, screen } from '@testing-library/react';

import HomePage from '@/app/(home)/page';

describe('Page: homepage', () => {
  it('should render homepage', () => {
    render(<HomePage />);

    const heading = screen.getByRole('heading', {
      name: /Hello World/i,
    });

    expect(heading).toBeInTheDocument();
  });

  it('should have valid snapshot', () => {
    render(<HomePage />);

    const homepage = screen.getByTestId('homepage');

    expect(homepage).toMatchSnapshot();
  });
});
