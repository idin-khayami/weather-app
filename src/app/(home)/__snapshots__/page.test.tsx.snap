// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page: homepage should have valid snapshot 1`] = `
<main
  data-testid="homepage"
>
  <h1>
    Hello World
  </h1>
</main>
`;
